### PR TITLE
feat: Added integration tests with postgres

### PR DESCRIPTION
**Pull request by saxenakshitiz**

### CODE REVIEW

File: query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPostgresQueriesTest.java [hunk 1]
```diff
@@ -127,6 +130,7 @@ public static void setup() throws Exception {
   @AfterAll
   public static void shutdown() {
     LOG.info("Initiating shutdown");
+    IntegrationTestServerUtil.shutdownServices();
```
Comment (kotharironak) [inline query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPostgresQueriesTest.java:133]: Do the same on Pinot tests too.

Comment (saxenakshitiz) [reply query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPostgresQueriesTest.java:138]: Already added. Didn't resolve the issue.

File: query-service-impl/src/main/java/org/hypertrace/core/query/service/postgres/converters/DefaultColumnRequestConverter.java [hunk 2]
```diff
@@ -138,46 +141,87 @@ private String convertFilterToString(
...
+    return isSimpleAttributeExpression(filter.getLhs())
+        && filter.getRhs().getValueCase().equals(LITERAL)
+        && context.isBytesColumnType();
+  }
+
+  /** Handles value conversion of a bytes expression based */
+  private void handleConversionForBytesExpression(
+      String lhs, String operator, Expression rhs, StringBuilder builder, Builder paramsBuilder) {
+    Value value = rhs.getLiteral().getValue();
+    if (value.getValueType().equals(ValueType.STRING)) {
```
Comment (kotharironak) [inline query-service-impl/src/main/java/org/hypertrace/core/query/service/postgres/converters/DefaultColumnRequestConverter.java:174]: nit: can we move to a separate function 
`handleEqualAndNotEqualOperatorForByteExpression`?

**Review by kotharironak — APPROVED** [review]
overall looks fine. we can handle follow up
- nit for refactoring handling of equal or not equal operator
- running both test pinot + postgres together

**Pull request merged**